### PR TITLE
Add new issue templates

### PR DESCRIPTION
**Pull request by snappizz**
turns out github has an extremely swanky feature for multiple issue templates for bug/feature reports! i split our current issue template into two and also trimmed down a bunch of comments so the templates are a little less overwhelming and cluttered.

### CODE REVIEW

**Comment by brianlheim** [comment]
Shouldn't this PR also delete the old issue template? (and same comment about [skip ci])

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+---
+
+<!-- Please see CONTRIBUTING.md for guidelines. -->
+
+## Environment
+
+* SuperCollider version:
+* Operating system:
+
```
Comment (brianlheim) [inline .github/ISSUE_TEMPLATE/bug_report.md]: leave the comment about Qt version/other relevant info, that does come in really handy and it's the easiest to forget

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+about: Create a report to help us improve
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+<!-- Please see CONTRIBUTING.md for guidelines. -->
+
+## Environment
```
Comment (brianlheim) [inline .github/ISSUE_TEMPLATE/bug_report.md]: again prefer ---- headings for same reason as PR template discussion

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+* SuperCollider version:
+* Operating system:
+
+## Steps to reproduce
+
+<!-- Use triple backticks ``` ``` to delimit code. Make sure to use a minimal example. -->
```
Comment (brianlheim) [inline .github/ISSUE_TEMPLATE/bug_report.md]: we can see how it plays out but I expect some people will misinterpret this and write triple backticks around inline code. the previous version of this was nice in that it demonstrated both the expected form and that you could use triple-backtick followed by 'supercollider' to get the syntax highlighting.

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+
+## Expected vs. actual behavior
+
+<!-- Paste error messages in entirety, delimited by ``` ```. Use gist if it's very long. -->
```
Comment (brianlheim) [inline .github/ISSUE_TEMPLATE/bug_report.md]: why remove the crash report prompt? this is valuable information and IME most people do not provide it unless prompted directly

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+
+<!-- Please see CONTRIBUTING.md for guidelines. -->
+
+## Motivation
+
+## Description of Proposed Feature
+
+## Plan for Implementation
+
+<!-- Who will implement this? How long will it take? -->
```
Comment (brianlheim) [inline .github/ISSUE_TEMPLATE/feature_request.md]: :+1:

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 4 above)
Comment (snappizz) [reply .github/ISSUE_TEMPLATE/bug_report.md]: restored!

**Comment by snappizz** [comment]
accidentally committed without `[skip ci]` again, had to cancel appveyor, so the CI status can be safely ignored here.

**Pull request merged**